web/modules/custom/bglobal_angular/components/voices_list/template.html

```diff
@@ -1,6 +1,18 @@
-<div *ngFor="let post of posts" data-ng-cloak class="voices-section grid-flip" style="width: 400px; padding: 10px;">
-  <!-- {{ i }} {{ getKeys(post) }} -->
-  <img [src] = "post.image.src" [alt] = "post.image.alt" style="width: 400px;" class="voice-section--img m-b-20 img-fluid" />
-  <p class="marginless-p"><strong><a [href] = "post.path" class="blog-header">{{ post.title }}</a></strong></p>
-  <div class="blog-intro" [innerHTML] = "post.content"></div>
+<!-- Reference:
+  /web/themes/custom/bglobal/source/_patterns/01-molecules/voices-list/voices-list.twig
```

Comment:
It would be better to handle voices_list changes in a separate PR, though we can do them here for now.